src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,36 @@
+package duke;
```

Comment:
Deadline class looks great!, the file is properly packaged, and all the imports are explicitly mentioned! 👍